Block/Adminhtml/Render/Version.php

```diff
@@ -74,8 +93,15 @@ public function render(AbstractElement $element)
             $html .= "</tr>";
 
         } catch (\Exception $e) {
+            $this->altapayLogger->addCriticalLog('Exception', $e->getMessage());
```

Comment:
No such function exists in either magento or this plugin

Reply:
That's correct! there was a logger folder bind with PHP-API but now that folder is removed but the method was still being there.
I have replaced Monolog with the standard Magento PSR Logger.